ci_check.sh

```diff
@@ -1,44 +0,0 @@
-#!/bin/bash
```

Comment:
Suggestion : Instead of removing this file and adding three more files we could have just added conditional statement for execution of the scripts based on the arguments recieved while executing the script `( ci_check.sh release)`

Reply:
let me check that and see if it will work for adding the E2E tests as well. 
In the next set of changes I have separated the installation and uninstallation by release into two different scripts.
Let me check if making the conditional script can solve the issue.

Reply:
@asifdxtreme you were right. I have tested this and made the changes. I will be pushing the changes shortly.